Assets/GothicVR/Scripts/UI/FontManager.cs

```diff
@@ -0,0 +1,34 @@
+using GVR.Phoenix.Interface;
+using GVR.Util;
+using System.Collections.Generic;
+using TMPro;
+
+namespace GVR.Manager
+{
+    public class FontManager : SingletonBehaviour<FontManager>
+    {
+        public Dictionary<string, TMP_FontAsset> fontDictionary = new Dictionary<string, TMP_FontAsset>(); // Dictionary to store font assets for each tag
+
+        public void Create()
+        {
+            // Populate the font dictionary with tag-fontAsset pairs
+            fontDictionary["MenuUI"] = GameData.I.GothicMenuFont;
```

Comment:
Is it really _MenuUI_? As there are only two fonts from WorldOfGothic, I would recommend we have:
1. UI
2. Subtitle
as fonts to reference.

Reply:
The 2 fonts used are called title font and ingame font. I would suggest we call them "Title" and "IngameText" both in the script and for the tags.

Reply:
Resolved.